management-api-agent-dse-6.8/src/main/java/com/datastax/mgmtapi/shim/DseAPI68.java

```diff
@@ -333,4 +334,8 @@ public RpcStatementShim makeRpcStatement(String method, String[] params) {
   public HintsService getHintsService() {
     return HintsService.instance;
   }
+
+  public void reloadTrustManager() throws Exception {
+    DseReloadableTrustManager.serverEncryptionInstance().reloadTrustManager();
```

Comment:
question: Is there a separate clientEncryptionInstance() that might need reload also?

Reply:
There is, but the implementation seems to be the same as `serverEncryptionInstance()`. @Miles-Garnsey it might be good to double check me on that.

Reply:
I don't want to get into this. It has nothing to do with getting CA rotation working and should be discussed elsewhere.